cfme/tests/infrastructure/test_vm_clone.py

```diff
@@ -57,6 +57,7 @@ def test_vm_clone(appliance, provider, clone_vm_name, create_vm):
     assert request_row.is_succeeded(method='ui'), msg
 
 
+@pytest.mark.rhv3
```

Comment:
The functionality of clone VM (create a VM from existing one) exist on RHV, but do not exist on CFME.
Not sure why this test is run for RHV provider.

Reply:
Great, thanks for explaining